xain/benchmark/aggregation/aggregation.py

```diff
@@ -8,6 +8,7 @@
     learning_rate,
     task_accuracies,
 )
+from xain.benchmark.aggregation.participant_hist import participant_history
```

Comment:
Please import the necessary module using the the same structure as previous lines do

Reply:
Updated.